extensions/ql-vscode/src/pure/sarif-utils.ts

```diff
@@ -173,3 +174,65 @@ export function parseSarifRegion(
 export function isNoLocation(loc: ParsedSarifLocation): loc is NoLocation {
   return 'hint' in loc;
 }
+
+// Some helpers for highlighting specific regions from a SARIF code snippet
+
+/**
+ * Checks whether a particular line (determined by its line number in the original file)
+ * is part of the highlighted region of a SARIF code snippet.
+ */
+export function shouldHighlightLine(
```

Comment:
Since these functions are nicely exported as part of the module now we can add unit tests for them 😸 I don't expect you to do that as part of this work/PR, just making a (cheeky) observation.

Reply:
Good idea 😁